kafka-client/src/main/java/dev/responsive/kafka/internal/db/inmemory/InMemoryKVTable.java

```diff
@@ -22,7 +24,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class InMemoryKVTable implements RemoteKVTable<Object> {
+// Note: this class doesn't actually use BoundStatements and applies all operations immediately,
+// we just stub the BoundStatement type so we can reuse this table for the TTD
+public class InMemoryKVTable implements RemoteKVTable<BoundStatement> {
```

Comment:
I suppose it's worth noting that this table implementation doesn't actually apply ttl in terms of retention, only in terms of filtering results. I figure this is fine for the TTD since such test should be short-lived, but good to note in case we ever want to do anything else with this class 🤷‍♀️